src/scripts/main.js

```diff
@@ -1,3 +1,24 @@
 'use strict';
 
-// write your code here
+const obj = [];
```

Comment:
```suggestion
const data = [];
```
Це масив. Чому тоді називається obj?

Reply:
Зрозумів тепер буду писати правильно